src/mongo/transport/README.md

```diff
@@ -1,40 +1,47 @@
+
```

Comment:
[Q] Do we need the extra line at the beginning?

Reply:
We do not. Thanks for catching it.